showcase.json

```diff
@@ -3480,5 +3480,26 @@
   "twitter": "https://twitter.com/Mapzlive",
   "youtube": "",
   "submittableId": "43548962"
+  },
+  {
+  "name": "🐲 Dragon Eyes 👀",
+  "description": "Provably fair dice rolling game as a testament that randomness can be obtained on-chain",
+  "website": "https://icdragon.xyz/",
+  "logo": "/img/showcase/icdragon_logo.png",
+  "screenshots": [
```

Comment:
Can you please upload a file with the same path?

Reply:
hi Jenn, 
yes, completed upload
please kindly continue

thanks

Reply:
Thank you, @vxyagr for the update! Nice to see another project from the Loka team